modules/update-upgrading-web.adoc

```diff
@@ -53,7 +53,7 @@ endif::openshift-origin[]
 
 . Select
 ifdef::within[]
-a version to update to,
+a new channel
 endif::within[]
 ifdef::between[]
 the highest available version
```

Comment:
It seems weird to have this flipping between a channel-change and a version-selection.  I would expect folks to bump their channel when their desired semantics changed (e.g. fast <-> candidate) or when their target minor changed (e.g. stable-4.7 -> stable-4.8).  And then to be selecting versions as they become available (and coincide with available maintenance windows or internal verification).  So two different use-cases and timescales.  And it is surprising to me that a single block of text could cover both with a within/between conditional.

Reply:
Rendered [*Updating a cluster by using the web console* section][1] (with highlighting by me for this new language):

![image](https://user-images.githubusercontent.com/209920/131371462-bd1e59a3-c055-451f-90c6-4c5a2749e9fe.png)

But changing the channel is telling the cluster "in the future, I'm interested in hearing about updates like...".  I was thinking that the channel change, if needed, would be happening in step 2.  In step 3 (or at least in some step in a chain that includes "Update to <product-version> in progress"), you'll need to actually select a target version.

[1]: https://deploy-preview-35589--osdocs.netlify.app/openshift-enterprise/latest/updating/updating-cluster?utm_source=github&utm_campaign=bot_dp#update-upgrading-web_updating-cluster

Reply:
I have reverted this content to its original form.